src/app/register/pages/register/register.component.ts

```diff
@@ -102,7 +102,7 @@ export class RegisterComponent implements OnInit, AfterViewInit {
             this.FormGroupStepA = this.prefillRegisterForm(
               this.platform.queryParameters
             )
-          } else if (session.oauthSession) {
+          } else if (session.oauthSession && Object.keys(platform.queryParameters).length !== 0) {
```

Comment:
Is this making you have an error?
Because if there is a oauthSession, the form can be prefilled with no problem even when there is no parameters